my-ansible-playbook.yml

```diff
@@ -0,0 +1,71 @@
+- name: Install To Do App on new web server
+  hosts: webserver
+  remote_user: ec2-user
+  vars_prompt:
+      - name: secret_key
+        prompt: Enter secret key
+        private: yes
+
+      - name: api_key
+        prompt: Enter the API key
+        private: yes
+
+      - name: token
+        prompt: Enter the token
+        private: yes
+
+      - name: board_id
+        prompt: Enter the board id
+        private: yes
+
+  tasks:
+  - name: Install git and Python 3
+    yum:
+        name: 
+          - git
+          - python3
+        state: present
+    become: yes
+
+  - name: Install Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3
+    args: 
+        creates: ~/.local/bin/poetry
+
+  - name: Create directory for the To-Do App code
+    ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        mode: '777'
+    become: yes
+        
+  - name: Get latest version of my code with Git
+    ansible.builtin.git: 
+        repo: 'https://github.com/WendiF/DevOps-Course-Starter.git'
+        dest: /opt/todoapp
+        version: exercise-4
+
+  - name: Install project dependencies
+    ansible.builtin.shell: |
+            source ~/.bash_profile
+            cd /opt/todoapp
```

Comment:
Rather than running a `cd` in your command, you can set the directory for the shell task with the `chdir` parameter.